src/main/java/org/zalando/nakadi/domain/SubscriptionBase.java

```diff
@@ -29,6 +29,7 @@
     private Set<String> eventTypes;
 
     @NotNull
+    @Size(min = 1, message = "must contain at least one character")
```

Comment:
What if users want to have consumer groups with a single char, like "1", "2", etc. They would be forced to do something like "001" which could be frustranting. I'm fine with single char.